packages/datatrak-web/src/api/mutations/useRequestProjectAccess.ts

```diff
@@ -26,10 +37,18 @@ export const useRequestProjectAccess = () => {
       });
     },
     {
-      onSuccess: () => {
-        queryClient.invalidateQueries({
-          queryKey: ['projects'],
-        });
+      onError(error: Error) {
+        if (options?.onError) options.onError(error);
+      },
+      onSettled() {
+        if (options?.onSettled) options.onSettled();
+      },
+      async onSuccess(response: ResponseBody) {
+        await Promise.all([
+          queryClient.invalidateQueries({ queryKey: ['me/countries'] }),
+          queryClient.invalidateQueries({ queryKey: ['projects'] }),
+        ]);
```

Comment:
Alternatively, this could’ve looked like this:

```tsx
await queryClient.invalidateQueries({
  predicate: query =>
    (['me/countries', 'projects'] as unknown[]).includes(query.queryKey[0]),
});
```

But in my testing this exact snippet doesn’t work correctly. I’m sure it’s my fault, but let me know if you think just using `Promise.all()` with two `invalijdateQueries`’s is fine

Reply:
I think in this case I would just simplify it to 
```
await queryClient.invalidateQueries({ queryKey: ['me/countries'] });
await queryClient.invalidateQueries({ queryKey: ['projects'] });
```

since there are only 2 to invalidate